cassandra-sink/src/test/resources/logback.xml

```diff
@@ -0,0 +1,10 @@
+<configuration>
```

Comment:
it was gone for a while, added it back in to get rid of all the logging during @BeforeClass